Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -810,7 +810,7 @@ public final class PackageBuilder {
             fileSystem: self.fileSystem,
             observabilityScope: self.observabilityScope
         )
-        let (sources, resources, headers, others) = try sourcesBuilder.run()
+        let (sources, resources, headers, ignored, others) = try sourcesBuilder.run()
```

Comment:
Adding another tuple member keeps diffs minimal for this PR but this is getting out-of-hand.  I plan to put up a PR to refactor the results coming back from the `TargetSourcesBuilder` but want to keep risk minimal to fix this bug.

Reply:
yea seems like this requires a bit of refactoring, even a struct to replace the tuple seems nicer